components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,64 @@
+const uservoice = require("../../reddit.app.js");
```

Comment:
You need to change this from `uservoice` to `reddit`.

Reply:
Changed.